package.json

```diff
@@ -166,7 +166,7 @@
     "url": "git://github.com/tidev/titanium_mobile.git"
   },
   "vendorDependencies": {
-    "node": "12.x || 14.x || 16.x"
+    "node": "12.x || 14.x || 16.x || 18.x || 20.x"
```

Comment:
@cb1kenobi I wonder if we should remove everything pre-16, but in a SDK 13.0.0

Reply:
https://github.com/tidev/titanium-sdk/pull/13807

Reply:
Please, please, please do not rip out anything or break things for now. As nice as the CLI v7 is, it's far away from a stable / tested release and will require some major upgrades to run stable. Fixing the warning in SDK 12.x should be the most convenient way for most users to continue working without updating their Node.js setup, while developers that have the latest LTS tooling can continue working as well, but without warnings.

Reply:
+1 for removing it! 
But I think we should add up to 20 in the package.json for 12.x just to get rid of the warning as there are no issues with using it - keeping the minimum as it is for now. 
This makes it easier for new users to just grab the latest node version from the page and not have a warning right away

Reply:
Sorry, yes, add 18 and 20 to SDK 12 to suppress the warning.

Definitely not suggesting we break anything anytime soon. Once v7 is merged and published as latest (it'll be a while), we can circle back and do a final v6 release that adds a guard around `vendorDependencies` and a message to update to v7. I want to avoid touching the CLI until v6 is in a maintenance branch and v7 is merged into `master`. The merge conflicts are going to be impossible to fix.